spyder_terminal/widgets/main_widget.py

```diff
@@ -15,7 +15,7 @@
 import os.path as osp
 
 # Third party imports
-from qtpy.QtCore import Signal, QTimer, Slot
+from qtpy.QtCore import Signal, QTimer, Slot, QProcess
```

Comment:
```suggestion
from qtpy.QtCore import QProcess, QTimer, Signal, Slot
```

* Just organizing this imports in alphabetical order.
* Also, please remove the import to the `subprocess` module in case it's not necessary anymore.